.circleci/config.yml

```diff
@@ -125,7 +125,9 @@ commands:
                 environment:
                   DD_TRACE_AGENT_URL: http://localhost:9126
 
-                command: "echo -p2.7,-p3.5,-p3.6,-p3.7,-p3.8,-p3.9 | tr ',' '\n' | circleci tests split | xargs ./scripts/ddtest riot -v run --exitfirst --pass-env -s '<< parameters.pattern >>'"
+                command: |
+                  mv .riot .ddriot
```

Comment:
why do we need this move?

Reply:
nvm, @Kyle-Verhoog helped clarify this offline. This is just for the docker tests, not the "bare metal" runs.

Reply:
We prebuild to .riot and then need to move to .ddriot for the docker tests because we mount .ddriot->.riot it in the docker test runner container